CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ All noteworthy changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) (as of Feb 2018)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/pseudomuto/protoc-gen-doc/compare/v1.3.2...master)
+## [Unreleased](https://github.com/pseudomuto/protoc-gen-doc/compare/v1.3.3...master)
```

Comment:
v1.3.3 hasn't been cut yet. I'd just add this change under the `## Changed` header in the _Unreleased_ section.

Reply:
Ah, OK, I wasn't sure what the release process was. Done, thanks.